src/jit/morph.cpp

```diff
@@ -2598,6 +2582,13 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
     bool            callIsVararg      = call->IsVarargs();
 #endif
 
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
+    // If fgMakeOutgoingStructArgCopy is called and
+    // copies are generated, hasStackArgCopy is set 
+    // to make sure to call evalArgsToTemp.
```

Comment:
Would it be useful to add to this comment *why* calling EvalArgsToTemps() is required? If not, please add that information to some other comment, below.

---
In reply to: [70518596](https://github.com/dotnet/coreclr/pull/6037#discussion_r70518596) [](ancestors = 70518596)